cmd/vcert/cmdCloudKeystores.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/Venafi/vcert/v5"
+	"github.com/Venafi/vcert/v5/pkg/endpoint"
+	"github.com/Venafi/vcert/v5/pkg/venafi/cloud"
+)
+
+var (
+	subCommandCloudKeystore = &cli.Command{
+		Name:  subCommandCloudKeystoreName,
+		Flags: provisionFlags,
+		Usage: "provision certificate from Venafi Platform to Cloud Keystore",
+		UsageText: `vcert provision cloudkeystore <Required Venafi Control Plane> <Options>
+
+   vcert provision cloudkeystore -k <VCP API key> --certificate-id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx --keystore-id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx --format json
```

Comment:
You may see that the spaces here looks odd, but as I changed this code into another file, I had to do it like this so in CLI, in order to respect its margin:
![Screenshot_20240523_100906](https://github.com/Venafi/vcert/assets/77068025/06b749b3-a159-4384-9fd4-c617c93ac91b)

Reply:
Enforce the use of the -p flag. It is the correct way to use the CLI now.